### PR TITLE
Update network-and-migration-planning.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Microsoft 365 network connectivity test tool to the list of tools to test network

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8241

File: microsoft-365/enterprise/network-and-migration-planning.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ To estimate the bandwidth, use the following steps:
 - [Microsoft Remote Connectivity Analyzer](https://go.microsoft.com/fwlink/p/?LinkId=517243): Tests connectivity in your Exchange Online environment.
     
 - Use the [Microsoft Support and Recovery Assistant for Office 365](https://diagnostics.office.com/#/Download?env=SOC) to fix Outlook and Office 365 problems. 
+
+- [Microsoft 365 network connectivity test tool](/microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool): Microsoft 365 network connectivity test tool.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/network-and-migration-planning.md]:
```suggestion
- [Microsoft 365 network connectivity test tool](/microsoft-365/enterprise/office-365-network-mac-perf-onboarding-tool): Tests Microsoft 365 network connectivity.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kelleyvice-msft Gentle reminder

**Pull request merged**